### PR TITLE
Remove test deprecation warnings

### PR DESCRIPTION
**Pull request by manuelpuyol**

### CODE REVIEW

File: test/components/blankslate_component_test.rb [hunk 1]
```diff
@@ -69,12 +69,12 @@ def test_renders_a_blankslate_component_with_an_icon_with_a_custom_size
 
   def test_renders_a_blankslate_component_with_an_image
     render_inline(Primer::BlankslateComponent.new(
-                    image_src: "file.svg",
+                    image_src: "/some_image",
```
Comment (manuelpuyol) [inline test/components/blankslate_component_test.rb:72]: was raising a `file.svg` not found warning

File: test/components/dropdown/menu_component_test.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 require "test_helper"
 
-class PrimerDropdownMenuComponentTest < Minitest::Test
+class Primer::Dropdown::MenuComponentTest < Minitest::Test
```
Comment (manuelpuyol) [inline test/components/dropdown/menu_component_test.rb:5]: duplicated name from `Primer::DropdownMenuComponent`

File: test/primer/classify_test.rb [hunk 3]
```diff
@@ -118,7 +118,7 @@ def test_underline
   end
 
   def test_color
-    assert_generated_class(nil,                  { color: "" })
+    refute_generated_class({ color: "" })
```
Comment (manuelpuyol) [inline test/primer/classify_test.rb:121]: was running `assert_nil`

**Review by srt32 — APPROVED** [review]
Thanks!

**Pull request merged**